### PR TITLE
Site getting started copy amends

### LINKED ISSUE #786: Getting started page copy updates

### PR DESCRIPTION
**Pull request by james-nash**
* Updates copy in getting started section to latest copy doc (v8)
* Simplifies heading styling to make it work consistently across browsers
* Makes accordion for font table _not_ use an `<h2>` as that creates an incorrect heading outline

closes #786

### CODE REVIEW

File: site/docs/getting-started/DesignCards.tsx [hunk 1]
```diff
@@ -11,42 +11,45 @@ type CardInfoType = { img: string; altText: string; content?: JSX.Element };
 const cardInfo: CardInfoType[] = [
   {
     img: Assets1,
-    altText: "figma components panel",
+    altText:
+      'Screenshot of Figma\'s assets panel displaying the "Salt: Components Light" library',
```
Comment (james-nash) [inline site/docs/getting-started/DesignCards.tsx:15]: @rufcha I took the liberty of coming up with some alt texts to describe these screenshots for screenreaders (required in order to meet accessibility criteria)

Please let me know if they ~look~ _sound_ OK to you :-)

You can suggest edits directly here in Github by:

1. Hovering over line number to get a + button and clicking it (or clicking and dragiing if you want to cover multiple lines)
2. Clicking the "Add suggestion" button above the comment filed that appears (the leftmost icon that looks like a file with + and - symbols on it)
3. Editng the text between in the "suggestion" block that gets inserted into the comment as desired
4. Submitting your comment

File: site/docs/getting-started/DesignCards.tsx [hunk 2]
```diff
@@ -70,8 +73,12 @@ const DesignCards = () => {
 };
 
 export const SoloImgCard = () => (
-  <div className={styles.designCardsContainer}>
-    <Card img={SelectIconImg} altText="figma icon library" />
+  <div className={styles.soloImg}>
```
Comment (JoanaMMoreira) [inline site/docs/getting-started/DesignCards.tsx:76]: Is this `soloImg` class being used?

File: site/docs/getting-started/DevelopFontsAccordion.module.css [hunk 3]
```diff
@@ -1,12 +1,3 @@
-.accordionWrapper {
-  --ifm-h2-font-size: var(--uitk-text-h4-fontSize);
-  --ifm-heading-font-weight: var(--uitk-text-h4-fontWeight);
```
Comment (JoanaMMoreira) [inline site/docs/getting-started/DevelopFontsAccordion.module.css:3]: I think this heading needs to be bold but may be best to check with @shey-v

File: site/src/css/custom.css [hunk 4]
```diff
@@ -236,37 +236,48 @@ ul.table-of-contents li > ul li {
...
 }
 
 .docs-page-content h2 {
+  margin-top: calc(var(--uitk-size-unit) * 3);
   margin-bottom: calc(var(--uitk-size-unit) * 2);
 }
 
-.docs-page-content > *:has(+ h2),
-.docs-page-content > *:not(h2):has(+ h3) {
+/* .docs-page-content > *:has(+ h2), */
```
Comment (JoanaMMoreira) [inline site/src/css/custom.css:252]: Can we delete the commented out styles?

File: site/docs/getting-started/DesignCards.tsx (hunk 2 above)
Comment (james-nash) [reply site/docs/getting-started/DesignCards.tsx:76]: No it's not. Nuking it :-)

**Review by JoanaMMoreira — APPROVED** [review]

**Review by rufcha — DISMISSED** [review]

File: site/docs/getting-started/develop.mdx [hunk 5]
```diff
@@ -137,13 +139,13 @@ function App() {
 export default App;
 ```
 
-Head over to our available components to view their respective APIs.
+Head over to our available [components](/components/) to view their respective APIs.
 
-### 5. (Optional) Play with labs components
+### 5. (Optional) Play with lab components
 
 Our `@jpmorganchase/uitk-labs` package is where we introduce new components that are in the initial stages of development. Once they’re stable, we move them over to the core package.
```
Comment (joshwooding) [inline site/docs/getting-started/develop.mdx:146]: Should be @jpmorganchase/uitk-lab

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: site/docs/getting-started/DesignCards.tsx (hunk 1 above)
Comment (joshwooding) [inline site/docs/getting-started/DesignCards.tsx:15]: I've heard two opinions on this phrasing.

1. Image of, Photo of etc. should not be used in alt text.
2. Screenshot of should be used for step by step instructions.

2. Seems to be coming mainly from educational (unis) sites.

Probably worth asking @maj-stella

Comment (james-nash) [reply site/docs/getting-started/DesignCards.tsx:15]: @joshwooding I've raised #823 for this

**Pull request merged**